src/main/java/com/company/CarService.java

```diff
@@ -0,0 +1,71 @@
+package com.company;
+
+import org.hibernate.SessionFactory;
+
+import javax.persistence.EntityManager;
+import java.util.Scanner;
+
+public class CarService {
+
+    private static final SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+    private static final EntityManager entityManager = sessionFactory.createEntityManager();
+
+
+    public void addCar() {
+        Scanner scanner = new Scanner(System.in);
+        Car car = new Car();
+        System.out.print("Insert brand: ");
+        car.setBrand(scanner.nextLine());
+        System.out.print("Insert model: ");
+        car.setModel(scanner.nextLine());
+        System.out.print("Insert color: ");
+        car.setColor(scanner.nextLine());
+        System.out.print("Insert license plate: ");
+        car.setLicensePlate(scanner.nextLine());
+        System.out.print("Insert car condition: working - w, damaged - d): ");
+        String carCond = scanner.nextLine();
+        if (carCond.equalsIgnoreCase("d")) {
+            car.setCarCondition(carCondition.DAMAGED);
+        } else {
+            car.setCarCondition(carCondition.WORKING);
+        }
+        System.out.print("Insert rental cost: ");
+        car.setRentalCost(scanner.nextBigDecimal());
+        scanner.nextLine();
+        entityManager.getTransaction().begin();
+        entityManager.persist(car);
+        entityManager.getTransaction().commit();
+
```

Comment:
Dodajmy te `entityManager.close()` bo chyba rzeczywiście powinny być na końcu transakcji.
https://stackoverflow.com/questions/10762974/should-jpa-entity-manager-be-closed